server/sonar-db-dao/src/main/java/org/sonar/db/qualityprofile/QualityProfileDao.java

```diff
@@ -228,8 +228,11 @@ public void deleteAllProjectProfileAssociation(String profileKey, DbSession sess
     return mapper(session).selectProjectAssociations(profileKey, nameQuery);
   }
 
-  private String sqlQueryString(@Nullable String query) {
-    return query == null ? "%" : "%" + query.toUpperCase(Locale.ENGLISH) + "%";
+  private static String sqlQueryString(@Nullable String query) {
```

Comment:
I agree that the new version is more readable (and you could use a constant for "%"), but how is this a quality flaw?
It's quite a matter of taste
another reason to make this change part of the same commit of all the other "fix quality flaws" commit